vars/folioHelm.groovy

```diff
@@ -198,9 +198,8 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
         moduleConfig <<
           [
             [eureka: [enabled         : true,
-                      sidecarContainer: [image: "${Constants.ECR_FOLIO_REPOSITORY}/folio-module-sidecar",
-                                         tag  : ns.getModules().allModules['folio-module-sidecar']]]]
-
+                      sidecarContainer: [ image: "${Constants.FOLIO_CI_REPOSITORY}/folio-module-sidecar",
+                                          tag  : ns.getModules().allModules['folio-module-sidecar'] ]]]
```

Comment:
Please consider using the following approach instead of the hardcode name of the repo. Something like this
"${repository_sidecar}/folio-module-sidecar"


  moduleConfig << [image         : [repository: "${repository}/${moduleName}",
                                    tag       : moduleVersion],
                   podAnnotations: [creationTimestamp: "\"${LocalDateTime.now().withNano(0).toString()}\""]]